windows/wintools_complete_install_verify/wintools_complete_install_verify.yml

```diff
@@ -128,6 +128,8 @@
 
         - name: "Reset base snapshot"
           include_tasks: ../../common/reset_base_snapshot.yml
+          vars:
```

Comment:
Linux has the same scenario. It's also can do the same change. Or create another merge request for Linux?

Reply:
No change for Linux. Thanks!